matrix/system_of_linear_equation_in_2_variables.py

```diff
@@ -0,0 +1,84 @@
+def calculate_x_and_y(eq1, eq2):
```

Comment:
Let's change the filename to `cramers_rule.py` so that it is easier to find and avoid future duplicates.
```suggestion
# https://en.wikipedia.org/wiki/Cramer%27s_rule


def cramers_rule(equation_1, equation2):
```